e2e/event_test.go

```diff
@@ -1,112 +1,106 @@
 package main
 
 import (
-	"context"
 	"testing"
-	"time"
-
-	"github.com/mesg-foundation/engine/hash"
-	"github.com/mesg-foundation/engine/protobuf/acknowledgement"
-	pb "github.com/mesg-foundation/engine/protobuf/api"
-	"github.com/mesg-foundation/engine/protobuf/types"
-	"github.com/stretchr/testify/require"
 )
 
 func testEvent(t *testing.T) {
-	stream, err := client.EventClient.Stream(context.Background(), &pb.StreamEventRequest{
-		Filter: &pb.StreamEventRequest_Filter{},
-	})
-	require.NoError(t, err)
-	acknowledgement.WaitForStreamToBeReady(stream)
+	t.SkipNow()
+	// TODO: cannot create an event so this test is not functional
```

Comment:
Should I simply delete this test?
We are not going to put back the create event endpoint, and the rest of the functions are tested in other tests.